wpilibcExamples/src/main/cpp/templates/sample/Robot.cpp

```diff
@@ -61,27 +61,30 @@ class Robot : public frc::SampleRobot {
 				"Auto Selector", kAutoNameDefault);
```

Comment:
This code doesn't work with the sendable chooser created in RobotInit. It should be like the old java code

Reply:
If the code that was commented out doesn't actually work, why is it included?

Reply:
There are two mutually exclusive ways to select auto represented in this template. The first (uncommented originally) is using sendable chooser. The second (commented out originally) is using the string chooser on the LabVIEW dashboard. In the previous version of the PR, it removed the sendable chooser code and uncommented the string chooser code, so it only supported one of the two ways.